migration-guide.md

```diff
@@ -0,0 +1,72 @@
+# Migrating from the Unleash Proxy to Edge
+
+Since Edge is built to be a near drop in replacement for the Unleash Proxy, this guide should take you through the differences between Edge and the Unleash Proxy, and how to migrate from the Unleash Proxy to Edge, and, in cases where the feature set isn't equivalent, how to achieve the same results in Edge. A full docker compose is provided [here](./examples/docker-compose.yml), that will spin up Edge, Unleash and Redis, to allow you to understand the configuration options in context. After starting the compose, you should be able to access the Unleash UI at `http://localhost:4242`, add a toggle and cURL the edge instance with:
+
+``` sh
+curl --location --request GET 'http://0.0.0.0:3063/api/client/features' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: default:development.unleash-insecure-api-token' \
+--data-raw ''
+```
+
+## Not supported
+
+- Custom Strategies are not supported in Edge today. If you need to use a custom strategy, you will need to use the Unleash Proxy. We believe that the overwhelming majority of custom strategies are better expressed as a set of Unleash Constraints. If you have a situation where this absolutely isn't the case, please raise this as an issue with details on what you're trying to achieve. We are looking into supporting custom strategies in Edge in the future.
```

Comment:
Can we add some doc links here? Also, because each bullet has a lot of text, it could maybe be good to instead break it up into subsections? But again, maybe not? I'll let you pick which one.

```suggestion
- [Custom Strategies](https://docs.getunleash.io/reference/custom-activation-strategies) are not supported in Edge today. If you need to use a custom strategy, you will need to use the Unleash proxy. We believe that the overwhelming majority of custom strategies are better expressed as a set of [strategy constraints](https://docs.getunleash.io/reference/strategy-constraints). If you have a situation where constraints **cannot** replace your strategy, please raise this as an issue with details on what you're trying to achieve. We are looking into supporting custom strategies in Edge in the future.
```